### PR TITLE
Soarian FHIR R4 Organization Documentation

### PR DESCRIPTION
**Pull request by DeepakGupta93**
Description
----
Review Request for Soarian FHIR R4 Organization Documentation

PR Checklist
----
- [ ] Screenshot(s) of changes attached before changes merged.
Entities_1
Organization_1
Organization_2
Organization_3
Organization_4

- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: layouts/soarian/r4/base/entities.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<h1 id="entities">Entities</h1>
+
+<p>Entities are used to track, relate, and locate things and people across the healthcare spectrum. They include things like Location, but also the Person resource. The Person resource is used to index a real world person providing a means to identify the same person across different healthcare organizations and roles.</p>
```
Comment (HRNWriter) [inline layouts/soarian/r4/base/entities.html:3]: In the second sentence, put a comma (,) after the phrase "The Person resource is used to index a real world person"

PRODOC-160857.

**Review by HRNWriter — COMMENTED** [review]
Please see my comments.

File: content/soarian/r4/base/entities/organization.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+#### Request
+
+    GET https://fhir-myrecord-sc.sandboxcerner.com/r4/3f2aca24-87f3-4eac-a6d7-1f75247e6b43/Organization?_id=HCU.1
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:SOARIAN_R4_ORGANIZATION_SEARCH_BY_ID) %>
+
+Note: The examples provided here are non-normative and replaying them in the public sandbox is not guaranteed to yield the results shown on the site
```
[the remark this answers is not in the document]
Comment (DeepakGupta93) [reply content/soarian/r4/base/entities/organization.md]: Added (.) at the end of the sentence.

File: content/soarian/r4/base/entities/organization.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## Retrieve by ID
+
+List an individual Organization resource by its ID:
```
[the remark this answers is not in the document]
Comment (DeepakGupta93) [reply content/soarian/r4/base/entities/organization.md]: changed it to "id"

File: layouts/soarian/r4/base/entities.html [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<h1 id="entities">Entities</h1>
+
+<p>Entities are used to track, relate, and locate things and people across the healthcare spectrum. They include things like Location, but also the Person resource. The Person resource is used to index a real world person, providing a means to identify the same person across different healthcare organizations and roles.</p>
+
+<h2 id="implemented-resources">Implemented Resources</h2>
```
[the remark this answers is not in the document]
Comment (DeepakGupta93) [reply layouts/soarian/r4/base/entities.html]: Added

File: content/soarian/r4/base/entities/organization.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+### Headers
+
+<%= headers fhir_json: true %>
+
+### Example Search by ID
+
+#### Request
+
+    GET https://fhir-myrecord-sc.sandboxcerner.com/r4/3f2aca24-87f3-4eac-a6d7-1f75247e6b43/Organization?_id=HCU.1
```
Comment (khansasultana) [inline content/soarian/r4/base/entities/organization.md:62]: Cross checking with Patient resource, we should ideally provide URLs which are public sandbox and are reachable. Patient resource has something like
https://fhir-myrecord-sc.cerner.com/r4/2f8f5ec1-b7b8-4be5-ae27-e308284dd9c1/Patient/A879904FD2FE4B2D90C89FDA84E1285F'

Update in all other instances likewise

File: lib/resources/example_json/soarian_r4_examples_organization.rb [hunk 6]
```diff
@@ -0,0 +1,340 @@
...
+    }.freeze
+
+    SOARIAN_R4_ORGANIZATION_SEARCH_BY_ID ||= {
+      "resourceType": 'Bundle',
+      "id": 'de340a64-b768-4d2a-ac8b-dc6496188539',
+      "type": 'searchset',
+      "timestamp": '2021-03-26T19:02:37-04:00',
+      "entry": [
+        {
+          "fullUrl": 'https://fhir-myrecord-sc.sandboxcerner.com/r4/3f2aca24-87f3-4eac-a6d7-1f75247e6b43/Organization/HCU.1',
```
Comment (khansasultana) [inline lib/resources/example_json/soarian_r4_examples_organization.rb:171]: Cross checking with Patient resource, we should ideally provide URLs which are public sandbox and are reachable. Patient resource has something like
https://fhir-myrecord-sc.cerner.com/r4/2f8f5ec1-b7b8-4be5-ae27-e308284dd9c1/Patient/A879904FD2FE4B2D90C89FDA84E1285F'

Update in all other instances likewise

**Review by khansasultana — APPROVED** [review]

**Review by vf056083 — APPROVED** [review]

**Review by VamsiMure — APPROVED** [review]

**Comment by PBKarthik** [comment]
After Publish
image
image
image

**Pull request merged**